### ISSUE TITLE
getting error while installing node jdbc

### ISSUE THREAD

**Issue by jmohanindia** [CraZySacX/node-jdbc#34]
C:\RestAPIs\WebAPI\node_modules\jdbc\node_modules\java>node "C:\Program Files\no
dejs\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp
.js" rebuild
gyp ERR! configure error
gyp ERR! stack Error: Command failed:
gyp ERR! stack     at ChildProcess.exithandler (child_process.js:647:15)
gyp ERR! stack     at ChildProcess.emit (events.js:98:17)
gyp ERR! stack     at maybeClose (child_process.js:755:16)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:822:
5)
gyp ERR! System Windows_NT 6.2.9200
gyp ERR! command "node" "C:\\Program Files\\nodejs\\node_modules\\npm\\node_modu
les\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\RestAPIs\WebAPI\node_modules\jdbc\node_modules\java
gyp ERR! node -v v0.10.29
gyp ERR! node-gyp -v v0.13.1
gyp ERR! not ok
npm ERR! java@0.4.6 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the java@0.4.6 install script.
npm ERR! This is most likely a problem with the java package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls java
npm ERR! There is likely additional logging output above.

npm ERR! System Windows_NT 6.2.9200
npm ERR! command "C:\\Program Files\\nodejs\\\\node.exe" "C:\\Program Files\\nod
ejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "jdbc"
npm ERR! cwd C:\RestAPIs\WebAPI
npm ERR! node -v v0.10.29
npm ERR! npm -v 1.4.14
npm ERR! code ELIFECYCLE
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     C:\RestAPIs\WebAPI\npm-debug.log
npm ERR! not ok code 0

**Comment by CraZySacX** [comment]
Make sure you have java on your PATH.  If that is not the issue, you may want to check out the issues at node-java.   node-jdbc depends on node-java and your install is failing to build node-java correctly.

**Comment by shlomihassan** [comment]
i have similar problem using jdbc with node v0.12.0 any idea?

**Comment by CraZySacX** [comment]
I'm not sure node-java is supported on 0.12.0 yet.  I'll try it out and see if I can reproduce the issue.

**Comment by christopherbalz** [comment]
It appears that it is supported:
```
"node": ">=0.10.0"
```
 - https://github.com/joeferner/node-java/blob/master/package.json#L12

**Comment by CraZySacX** [comment]
Can you attach the output here of running ```npm install node-java``` from a command prompt.  It would also help to have the output of ```java -version```

**Comment by hpjcu** [comment]
Hi, I am having the same issue, and here is a full snapshot of the "npm install java" plus the version. I have tried (and reinstalled) Java 1.8, and 1.7, and Python, and have also searched for the missing Microsoft file called out in the jdbc install error. I have also checked all my paths as well - all files are found from the command line (node, npm, java and Python.exe).

I get the exact error installing jdbc-metadata (I guess expected). I have no issues installing something like Asynch.

Appreciate any help, leaving the country tomorrow and need to get this installed before I go.

Thanks!!
Juan at HP

![jdbc_install_error](https://cloud.githubusercontent.com/assets/12216220/7551557/6a2b674c-f652-11e4-9076-9e257013d866.JPG)
![java_install_error](https://cloud.githubusercontent.com/assets/12216220/7551558/6a2edb84-f652-11e4-9b2e-83ec03541628.JPG)
